lib/internal/test_runner/test.js

```diff
@@ -172,10 +172,13 @@ class Test extends AsyncResource {
 
   constructor(options) {
     super('Test');
-
-    let { fn, name, parent, skip } = options;
+    let { fn, name, parent, skip, testNamePatterns = [] } = options;
     const { concurrency, only, timeout, todo, signal } = options;
 
+    if (testNamePatternsFromCLI) {
+      testNamePatterns = testNamePatterns.concat(testNamePatternsFromCLI);
```

Comment:
I still think we should not merge these.

Reply:
Consider using `ArrayPrototypePushApply` instead of `.contact`